components/Timeline.jsx

```diff
@@ -3,7 +3,7 @@ import { TIMELINE_DATA } from "../data/timeline";
 import { UnderlineTypes } from "../utils/underlineType";
 import Heading from "./Heading";
 
-const TIMELINE_CONTAINER = "bg-white flex flex-col";
+const TIMELINE_CONTAINER = "bg-white flex flex-col lg:h-0 lg:min-height-full";
 const PURPLE_SIDE_LINES_DIR = "/svgs/timeline/left_side_lines.svg";
 
 const Timeline = () => {
```

Comment:
Before the mobile breakpoint, the images look a little weird. Maybe use `md:` breakpoint as well?

<img width="498" alt="Screen Shot 2022-08-17 at 10 56 34 PM" src="https://user-images.githubusercontent.com/47438886/185304751-9ec48777-7f91-4107-8d26-ae76e9e67300.png">

Reply:
Will do

Reply:
Fixed this